Dockerfile.notebook

```diff
@@ -0,0 +1,40 @@
+# Build Pyradiomics inside the Jupyter Datascience Notebook
+
+FROM jupyter/datascience-notebook
+
+MAINTAINER "Daniel Blezek" daniel.blezek@gmail.com
+
+USER root
+ADD . /root/pyradiomics
+# Install in Python 3
+RUN /bin/bash -c "source activate root \
+    && cd /root/pyradiomics \
+    && python -m pip install --no-cache-dir -r requirements.txt \
+    && python setup.py install"
+# Python 2
+RUN /bin/bash -c "source activate python2 \
+    && cd /root/pyradiomics \
+    && python -m pip install --no-cache-dir -r requirements.txt \
+    && python setup.py install"
+
+# Install sample data and notebooks
+ADD data/ /home/jovyan/work/example_data/
+ADD bin/Notebooks/RadiomicsExample.ipynb /home/jovyan/work/
+ADD bin/Notebooks/FeatureVisualization.ipynb /home/jovyan/work/
+ADD bin/Notebooks/FilteringEffects.ipynb /home/jovyan/work/
+ADD bin/Params.yaml /home/jovyan/work/
+
+# Make a global directory and link it to the work directory
+RUN mkdir /data
+RUN ln -s /data /home/jovyan/work/data
```

Comment:
Instead of creating a custom image for this, I would suggest to:
* use the official jupyter images
* ask the user to simply clone the repository container the notebooks
* run the jupyter image mounting that folder
* Installation of Pyradiomics and its dependency would be step 1 of the notebook

Doing so, we would have one less image to maintain.